### DISCUSSION TITLE
terrain edge sand

### DISCUSSION THREAD

**Discussion by 733737** [OpenRCT2/OpenRCT2#18129] [Feature requests] [answered]
Right now there is no good terrain edge to match the sand surface, I propose we create a custom sand terrain edge that is similar to the sand surface that would be a better fit for a sandy style park

**Comment by 733737** [comment, marked as answer]
there is an rct1 sandstone edge that I think is good, so it seems that for now at least you need rct1 installed and linked

**Reply by jjpb44** [reply]
Could you add images to see the comparison of the edges?
